react-app/src/app/screens/Game/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+
+import Board from './components/Board';
+import style from './styles.scss';
+
+class Game extends Component {
+  state = {
+    history: [{ squares: Array(9).fill(null) }],
+    stepNumber: 0,
+    xIsNext: true
+  };
+
+  handleClick = i => {
+    const history = this.state.history.slice(0, this.state.stepNumber + 1);
+    const current = history[history.length - 1];
+    const squares = current.squares.slice();
+    if (this.calculateWinner(squares) || squares[i]) {
+      return;
+    }
+    squares[i] = this.state.xIsNext ? 'X' : 'O';
+    this.setState({
+      history: history.concat([{ squares }]),
+      stepNumber: history.length,
+      xIsNext: !this.state.xIsNext
+    });
+  };
+  jumpTo = step => {
+    this.setState({
+      stepNumber: step,
+      xIsNext: step % 2 === 0
+    });
+  };
+  calculateWinner = squares => {
```

Comment:
This method is totally independent from the current component. You can extract it to an utils.js file in this same folder.